src/core/server/saved_objects/service/lib/repository.ts

```diff
@@ -448,11 +448,11 @@ export class SavedObjectsRepository {
     }
 
     let kueryNode;
+
     try {
-      kueryNode =
-        filter && filter !== ''
-          ? validateConvertFilterToKueryNode(allowedTypes, filter, this._mappings)
-          : null;
+      if (filter && filter) {
```

Comment:
Now that this has been rewritten a bit, is `(filter && filter)` doing anything that couldn't be accomplished with just `(filter)`?

Reply:
👍